internal/host/transition.go

```diff
@@ -390,22 +390,6 @@ func If(id stringer) stateswitch.Condition {
 	return ret
 }
 
-func (th *transitionHandler) IsPreparingTimedOut(sw stateswitch.StateSwitch, args stateswitch.TransitionArgs) (bool, error) {
-	sHost, ok := sw.(*stateHost)
-	if !ok {
-		return false, errors.New("IsPreparingTimedOut incompatible type of StateSwitch")
-	}
-	params, ok := args.(*TransitionArgsRefreshHost)
-	if !ok {
-		return false, errors.New("IsPreparingTimedOut invalid argument")
-	}
-	var cluster common.Cluster
-	if err := params.db.Select("status").Take(&cluster, "id = ?", sHost.host.ClusterID.String()).Error; err != nil {
-		return false, err
-	}
-	return swag.StringValue(cluster.Status) != models.ClusterStatusPreparingForInstallation, nil
-}
-
 func (th *transitionHandler) HasClusterError(sw stateswitch.StateSwitch, args stateswitch.TransitionArgs) (bool, error) {
```

Comment:
I don't think that this function is used anymore. maybe you should remove it.

Reply:
OK. Next PR